src/App.tsx

```diff
@@ -39,6 +42,28 @@ function App() {
 
   // function seekSong()
 
+  useEffect(() => {
```

Comment:
add a comment explaining the need for this useEffect. It looks like it exists just to perform a redirect in the case that you receive a 401 from spotify? why do we need to do this?